packages/node_modules/@ciscospark/react-container-call-incoming/src/index.js

```diff
@@ -0,0 +1,128 @@
+import {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {Howl} from 'howler';
+
+import {
+  callIncoming,
+  listeningForIncomingCalls,
+  removeCallIncoming
+} from '@ciscospark/redux-module-media';
+
+import {
+  NOTIFICATION_TYPE_CALL,
+  createNotification
+} from '@ciscospark/react-container-notifications';
+
+import Ringtone from './media/Ringtone.mp3';
+
+
+export class IncomingCall extends Component {
```

Comment:
I don't know why, but for some reason I don't like creating React components that don't render elements. That just seems wrong to me since React was really made to render DOM elements efficiently. I could see most of this as a redux module as most of this is state management.
The ringtone does make sense being it's own component. Can that be pulled out into it's own presentation component so that it can be reused?